db/src/main/scala/slick/syntax/DBIOActionDbSyntax.scala

```diff
@@ -12,4 +13,7 @@ trait DBIOActionRunSyntax {
 
 final class DBIOSyntaxOps[F[_], R, +S <: NoStream, -E <: Effect](val x: DBIOAction[R, S, E]) extends AnyVal {
   def run(implicit F: Async[F], slickDb: SlickDb): F[R] = slickDb.run(x)
+
+  def runOrDefault(default: R)(implicit F: Async[F], slickDb: SlickDb): F[R] =
+    slickDb.run(x).orElse(slickDb.run(x)).handleError(_ => default)
```

Comment:
I don’t understand why an attempt to re-execute the transaction is required, but without it only 98-99 deployments out of 100 are added correctly, and accordingly the test fails.